packages/paste-core/components/avatar/package.json

```diff
@@ -24,6 +24,10 @@
     "clean": "rm -rf ./dist",
     "tsc": "tsc"
   },
+  "dependencies": {
+    "@types/react-is": "^17.0.3",
```

Comment:
Note: I ended up needing to include the `@types/react-is` package as well as `react-is`